### DISCUSSION TITLE
Question about oxidation

### DISCUSSION THREAD

**Discussion by JianAtSeer** [vdemichev/DiaNN#741] [Q&A]
Hi Vadim:
I got question. We noticed in the gui, when u click on the methionine oxidation modification as variable mod,
in the corresponding command-line, the flag "--monitor-mod" is not added.  Note that this flag is added for if
we click on the other mods.  I am wondering why? My understanding is the "monitor-mod" turn on the PTM-specific
scoring.  Is this scoring not recommended for Ox(M) modification? Thanks

**Comment by vdemichev** [comment]
Yes, Ox(M) is a common technical artifact and is thus usually not of biological interest, hence by default PTM scoring is disabled for it.

**Comment by JianAtSeer** [comment]
Hi Vadim:
Thanks for your quick reply, I understand the artifact part.  But from technical perspective, is the scoring NOT applicable for this modification?  If I am interested in these peptides with these mods, should I still turn on the PTM-scoring for better control of the FDR?
